cmd/assetsvc/server/handler.go

```diff
@@ -297,30 +276,6 @@ func getChartVersionSchema(w http.ResponseWriter, req *http.Request, params Para
 	w.Write([]byte(files.Schema))
 }
 
-// listChartsWithFilters returns the list of repos that contains the given chart and the latest version found
-func listChartsWithFilters(w http.ResponseWriter, req *http.Request, params Params) {
-	namespace, repo, _, paramErr, err := extractDecodedNamespaceAndRepoAndVersionParams(params)
-	if err != nil {
-		handleDecodeError(paramErr, w, err)
-		return
-	}
-	cq := extractChartQueryFromRequest(namespace, repo, req)
-
-	pageNumber, pageSize := getPageAndSizeParams(req)
```

Comment:
This was the only non-test usage of the page number, and it's a handler for an API that we no longer used (replaced by the helm plugin), so just removed it.